apps/atlas/src/components/sidebars/MenuSideBar.tsx

```diff
@@ -44,135 +91,25 @@ export const MenuSideBar = () => {
           mainMenu ? '' : 'translate-y-full hidden sm:transform-none sm:block'
         }`}
       >
-        <div className="flex flex-col place-items-center ">
-          <IconButton
-            className={`${buttonClasses} ${
-              selectedPanel === 'realm' ? 'bg-gray-700' : ''
-            }`}
-            aria-label="Realms"
-            variant="unstyled"
-            texture={false}
-            onClick={() => {
-              togglePanelType('realm');
-            }}
-            icon={<Castle className={`${iconClasses}`} />}
-            size="lg"
-          />
+        {menus.map((menu) => (
+          <Link href={getPageHref(menu.page)} key={menu.page}>
```

Comment:
Back button navigation doesn't work as I expected, going from Realms List to Realms Detail and then back. One solution is to handle `onClick` a same-page change to router query params, with `shallow:true` option of the Link. (to avoid Next re-rendering).

Reply:
nvm wasn't looking at preview, oops